nodes/clusters/nodes-cluster-limit-ranges.adoc

```diff
@@ -0,0 +1,39 @@
+:context: nodes-cluster-limit-ranges
+[id="nodes-cluster-limit-ranges"]
+= Setting Limit Ranges
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+By default, containers run with unbounded compute resources on an {product-title} 
+cluster. With limit ranges, you can restrict resource consumption for specific 
+objects in a project: 
+
+* Pods and containers. You can set minimum and maximum requirements for CPU and
+memory for Pods and their containers.
+* Image streams. You can set limits on the number of images and tags in an 
+imagestream object.
```

Comment:
I've dropped my resistance to capitalizing objects, so this probably needs to be ImageStream:

https://docs.openshift.com/container-platform/4.5/rest_api/image_apis/imagestream-image-openshift-io-v1.html